src/screens/Home/HomeV2.res

```diff
@@ -37,13 +41,20 @@ module HomePageHorizontalStepper = {
         <div className="flex flex-col gap-2.5 w-full" key={index->Int.toString}>
           <div className="flex items-center gap-2">
             <span
-              className={`h-6 w-6 flex items-center justify-center border-2 rounded-md font-semibold ${index->getStepperStyle} ${getTextStyle}`}>
-              {(index + 1)->Int.toString->React.string}
+              className={`h-6 w-7 flex items-center justify-center border-1 rounded-md font-semibold ${index->getStepperStyle} ${getTextStyle}`}>
+              <UIUtils.RenderIf condition={index < step}>
+                <Icon name="check" size=12 customIconColor="#006DF9" />
```

Comment:
move this color code tailwind file

Reply:
we are unable to access the tailwind config file here, so we need to use direct color here